hasura/metadata/databases/default/tables/public_app_metadata.yaml

```diff
@@ -188,6 +188,9 @@ update_permissions:
   - role: reviewer
     permission:
       columns:
+        - logo_img_url
```

Comment:
also likely not needed, `service` role performs these updates

Reply:
aligned